### DISCUSSION TITLE
Feature request: Don’t show sidebar in compact mode when link is opened

### DISCUSSION THREAD

**Discussion by aohoyd** [zen-browser/desktop#2032] [Ideas]
Currently in compact mode sidebar is shown every time when new tab is opened. I think it would be more convenient not to show it every time

**Comment by ficd0** [comment]
Under Settings>Look and Feel>Theme Settings> Show in compact view; there is a checkbox `Briefly make the toolbar popup when switching or opening tabs in compact mode` where you can disable this behavior.
